projects/plugins/boost/app/assets/src/js/features/performance-history/lib/hooks.ts

```diff
@@ -18,6 +18,14 @@ const periodsSchema = z.object( {
 const performanceHistoryDataSchema = z
 	.object( {
 		periods: z.array( periodsSchema ),
+		annotations: z
+			.array(
+				z.object( {
+					timestamp: z.number(),
+					text: z.string(),
+				} )
+			)
+			.optional(),
```

Comment:
The PHP schema for `annotations` is nullable and if PHP returns null, this will make JS sad. Though, I don't know when that might happen.

Either way, I noted it just to be safe, as we ran into this recently - https://github.com/Automattic/jetpack/pull/34932.

Reply:
Thanks, I have made them not optional in the context of boost since we can actually control the back-end data. https://github.com/Automattic/jetpack/pull/34978/commits/e55a9505311af62d422aeae56549e9d78fbe3919